test/DynamoCoreTests/Graph/Nodes/NodeAttributesTests.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Linq;
+using CoreNodeModels.Properties;
+using Dynamo.Graph.Nodes;
+using NUnit.Framework;
+
+namespace Dynamo.Tests
+{
+    /// <summary>
+    /// This file contains tests for the classes in the Attributes file.
+    /// </summary>
+    [TestFixture]
+    class NodeAttributesTests
+    {
+        [Test]
+        [Category("UnitTests")]
+        public void NodeSearchTagsAttributeConstructorTest()
+        {
+            //resourceType is null
+            string tagsID = "tagsID";
+            Type resourceType = null;
+
+            Assert.Throws<ArgumentNullException>(() => _ = new NodeSearchTagsAttribute(tagsID, resourceType));
+        }
+
+        [Test]
+        [Category("UnitTests")]
+        public void NodeDescriptionAttributeConstructorTest()
+        {
+            //resourceType is null
+            Assert.Throws<ArgumentNullException>(() => _ = new NodeDescriptionAttribute("", null));
+
+            //resourceType is valid
+            string descriptionResourceID = "WatchNodeDescription";
+            Type resourceType = typeof(Resources);
+
+            var attr = new NodeDescriptionAttribute(descriptionResourceID, resourceType);
+            Assert.AreEqual("Visualize the output of node.", attr.ElementDescription);
```

Comment:
Where is `"Visualize the output of node."` defined?

Reply:
That string is defined in a resource, that is why I use it on the test as a valid input. Now that you mention it, It's best that I change that to the resource: CoreNodeModels.Properties.Resources.WatchNodeDescription